src/main/java/seedu/duke/entities/Order.java

```diff
@@ -24,12 +24,22 @@ public int getOrderLength() {
         return this.dishes.size();
     }
 
-    public double getTotalPrice() {
+    public static double getTotalPrice() {
         double totalPrice = 0;
-        for (int i = 0; i < this.dishes.size(); i++) {
-            totalPrice += this.dishes.get(i).getPrice();
+        for (int i = 0; i < dishes.size(); i++) {
+            totalPrice += dishes.get(i).getPrice();
         }
         return totalPrice;
     }
 
+    public static void printOrder() {
+        if (dishes.size() == 0) {
+            System.out.println("No orders!");
```

Comment:
if you want to add assert statment, just add one `assert dishes.size() == 0` here or smth

Reply:
below the if statement

Reply:
OK, thank you.